pkg/kubelet/container/helpers.go

```diff
@@ -365,3 +366,43 @@ func MakePortMappings(container *v1.Container) (ports []PortMapping) {
 	}
 	return
 }
+
+// ConvertCadvisorFsInfo converts cadvisorapiv2.FsInfo to runtimeapi.FsInfo.
+func ConvertCadvisorFsInfo(fsInfo cadvisorapiv2.FsInfo) runtimeapi.FsInfo {
+	runtimeFsInfo := runtimeapi.FsInfo{
+		Device:         fsInfo.Device,
+		Path:           fsInfo.Mountpoint,
+		CapacityBytes:  &runtimeapi.UInt64Value{Value: fsInfo.Capacity},
+		AvailableBytes: &runtimeapi.UInt64Value{Value: fsInfo.Available},
+		UsedBytes:      &runtimeapi.UInt64Value{Value: fsInfo.Usage},
+	}
+
+	if fsInfo.Inodes != nil {
+		runtimeFsInfo.InodesCapacity = &runtimeapi.UInt64Value{Value: *fsInfo.Inodes}
+	}
+	if fsInfo.InodesFree != nil {
+		runtimeFsInfo.InodesAvailable = &runtimeapi.UInt64Value{Value: *fsInfo.InodesFree}
+	}
+
+	return runtimeFsInfo
+}
+
+// ConvertRuntimeFsInfo converts runtimeapi.FsInfo to cadvisorapiv2.FsInfo.
+func ConvertRuntimeFsInfo(imageFsInfo runtimeapi.FsInfo) cadvisorapiv2.FsInfo {
```

Comment:
I think we should probably convert to runtimeapi.FsInfo everywhere rather than continuing to use the cAdvisor API, which would remove the need for this method. I'm ok with that being a cleanup PR, but maybe leave a TODO here?